src/components/community/kristen-kinnearohlmann.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+export default {
+    id: 'kristen-kinnearohlmann',
+    name: 'Kristen Kinnear-Ohlmann',
+    twitter: 'https://twitter.com/kinnear_ohlmann',
+    website: '',
+    linkedin: 'https://www.linkedin.com/in/kristenkinnearohlmann/',
+    github: 'https://github.com/kristenkinnearohlmann',
+    bio: () => (
+
```

Comment:
You need a bio, Kristen!

Reply:
Hey Kristen, don't forget to come back through again and add a bio?